spec/services/moderator/banish_user_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+
+RSpec.describe Moderator::BanishUser, type: :service, vcr: {} do
+  let(:user) { create(:user, :with_complete_profile) }
+  let(:user_without_email) { create(:user, email: nil) }
+  let(:admin) { create(:user, :super_admin) }
+
+  describe "banish_user" do
+    it "unsubscribe user from newsletters if user has email address" do
+      allow(user).to receive(:unsubscribe_from_newsletters)
+      described_class.call_banish(user: user, admin: admin)
+      expect(user).to have_received(:unsubscribe_from_newsletters).once
+    end
+
+    context "with user profile info" do # Split into 2 to make rubocop happy
+      it "removes user profile details" do
+        expect do
+          described_class.call_banish(user: user, admin: admin)
+        end.to change(user, :twitter_username).to(nil).and change(user, :github_username).to(nil).
+          and change(user, :summary).to("").and change(user, :location).to("").
+          and change(user, :education).to("").and change(user, :employer_name).to("").
+          and change(user, :employer_url).to("").and change(user, :employment_title).to("").
+          and change(user, :mostly_work_with).to("").and change(user, :currently_learning).to("").
+          and change(user, :currently_hacking_on).to("").and change(user, :available_for).to("").
+          and change(user, :email_public).to(false)
```

Comment:
I would use parenthesis here, since you're calling a `and()` method and not the `and` operator which is a valid Ruby operator. Easier to understand for the casual reader. 

Same in the other examples :)

Reply:
@rhymes Fixed.